otelcol/testdata/components-output.yaml

```diff
@@ -41,3 +41,6 @@ extensions:
       module: go.opentelemetry.io/collector/extension/extensiontest v1.2.3
       stability:
         extension: Stable
+providers:
+    - name: nop
```

Comment:
One last thing, should we call this `scheme` instead of `name`?

Reply:
Sure! I changed that.